[HW]1072946

yeepii opened the pull request:
要在PDF和README.md兩個之間切換有點不方便，所以整合到README.md中，
而且 git@github.com/YOUR_ACCOUNT/ncyu-2021有打錯，github.com和帳號中間應該是 " : " ，而不是" / "。
Signed-off-by: 1072946 <s1072946@mail.ncyu.edu.tw>

jrjang commented:
Hi @yeepii ,

1. commit message 格式有誤
  1.1 commit summary 的 [HW] 與 學號 之間要有空白
  1.2 以下描述是寫在 commit message 中
```
要在PDF和README.md兩個之間切換有點不方便，所以整合到README.md中，
而且 git@github.com/YOUR_ACCOUNT/ncyu-2021有打錯，github.com和帳號中間應該是 " : " ，而不是" / "。
```

jrjang commented on README.md:
```diff
@@ -1,6 +1,8 @@
 Example 6:
 
-1. 依照投影片描述做修改
+1. -先輸入git clone git@github.com:YOUR_ACCOUNT/ncyu-2021(或是git clone git clone https://github.com/YOUR_ACCOUNT/ncyu-2021)
```
請分項目將投影片所需步驟完整寫在 README.md. 例如:

1. 下載自己帳號的 ncyu-2021 project 至 ncyu-2021-ex6 資料夾中
[提示] ncyu-2021 project 位置是 `git@github.com:YOUR_ACCOUNT/ncyu-2021`
2. ...
3. ...

注意:
1. 中英文之間要有空白
2. "指令"或"程式碼"需要使用 github Quoting code 格式 (https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code)

Commit:
[HW] 1072946

要在 PDF 和 README.md 兩個之間切換有點不方便，所以整合到 README.md 中，
而且 git@github.com/YOUR_ACCOUNT/ncyu-2021 有打錯， github.com 和帳號中間應該是 : ，而不是  / 。

Signed-off-by: 1072946 <s1072946@mail.ncyu.edu.tw>
